plextraktsync/trakt_list_util.py

```diff
@@ -44,6 +44,11 @@ def __init__(self, username, listname):
             ]
             self.trakt_items = dict(zip(prelist, count(1)))
 
+    @property
+    def plex_items_sorted(self):
+        _, items = zip(*sorted(dict(reversed(self.plex_items)).items()))
```

Comment:
There should be a `rank` defined for each List Item.
https://trakt.docs.apiary.io/#reference/users/list-like/get-items-on-a-personal-list
Possible to sort by this? 

Also, it seems the refactor you implemented here #1183 removed the ability to create new Plex playlists. It only updates existing ones.

Reply:
No. Read again. And comment in context, i.e. that PR if you have anything to add.

Reply:
I'll add a mental note to sort by rank TODO. currently this is just same logic just shuffled around (not changed in this PR)

Reply:
respecting trakt list rank was added in https://github.com/Taxel/PlexTraktSync/pull/58

Reply:
I don't understand that changeset fully, i try to decode:
- `plex_movies.append((rank, plex_movie))` the first tuple item is rank
- `sorted(self.plex_movies)` by default sorts by tuple ascending: `(rank, plex_movie)`

but somewhere else, the sorting changed to dict there.

but anyway, as I did not change the logic in this PR, the trakt list rank is still here?

Reply:
It was just a reminder because you said :

> this just seems to be some by some arbitrary order

Reply:
I said that because I don't know what this code does:
`_, items = zip(*sorted(dict(reversed(self.plex_items)).items()))`

added description now: 69870acd